docs/configuration/overview/admin.rst

```diff
@@ -9,13 +9,16 @@ Administration interface :ref:`operations documentation <operations_admin_interf
 
   {
     "access_log_path": "...",
+    "profile_path": "...",
     "port": "..."
   }
 
 access_log_path
   *(required, string)* The path to write the access log for the administration server. If no
   access log is desired specify '/dev/null'.
-
+profile_path
+  *(optional, string)* The cpu profiler output path for the administration server. If no profile
+  path is specified, the default is '/var/log/envoy/envoy.prof'.
 address
```

Comment:
Sorry I meant please keep a blank line between attributes as per the rest of the documentation.